package.json

```diff
@@ -160,7 +160,7 @@
     "mockery": "2.1.0",
     "nock": "10.0.6",
     "node-wp-i18n": "1.2.3",
-    "prettier": "github:Automattic/wp-prettier#wp-prettier-1.16.4",
+    "prettier": "https://github.com/Automattic/wp-prettier/releases/download/wp-1.16.4/wp-prettier-1.16.4.tgz",
```

Comment:
What's the reasoning behind that change?

Reply:
It's an optimized bundle of our prettier flavor that provides the bin without any excess. Details: https://github.com/Automattic/wp-calypso/pull/29987